rb/Gemfile.lock

```diff
@@ -17,7 +17,6 @@ GEM
       safe_yaml (~> 1.0.0)
     diff-lcs (1.3)
     ffi (1.9.25)
-    ffi (1.9.25-x64-mingw32)
```

Comment:
Can you please revert this change? It is added when installing gems on Windows so, once merged, it will be added back on the next `bundle install` run on Windows.

Reply:
Ok -- is there a specific reason Gemfile.lock is even checked in? or possibly added and then gitignored if it being updated is a problem?

Reply:
Frankly, I don't remember why exactly it was added. Probably because we used cached gems and if there is no Gemfile - it pulls more recent versions. I haven't checked that, so maybe we're safe to remove it altogether?

Reply:
:point_up: Just for reference, this is the reason. I remember chatting with simon about this a while ago when I did a minor tweak